### PR TITLE
Only shows components in the force app directory

### PR DESCRIPTION
**Pull request by LisandroFernandezSF**
### What does this PR do?
It catches and shows the error thrown by @salesforce/lwc-dev-server

### What issues does this PR fix or reference?
@W-8231933@

### CODE REVIEW

File: packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts [hunk 1]
```diff
@@ -147,6 +148,10 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
         if (data.toString().includes(errorHints.INACTIVE_SCRATCH_ORG)) {
           this.errorHint = errorHints.INACTIVE_SCRATCH_ORG;
         }
+        if (data.toString().includes('No \'lwc\' directory found')) {
```
Comment (sfsholden) [inline packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts:151]:
```suggestion
        if (data.toString().includes(errorHints.LWC_FOLDER_NOT_FOUND)) {
```

File: packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts [hunk 2]
```diff
@@ -147,6 +148,10 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
...
+          errorCode = 135;
+          this.errorHint = errorHints.LWC_FOLDER_NOT_FOUND;
```
Comment (sfsholden) [inline packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts:153]: I think you're okay to go with either the errorCode or the errorHint approach, both aren't needed though. The reason why errorHint was originally added was to further clarify the server startup reason if we had additional information from stderr (provided from the lwc dev server).

File: packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts [hunk 3]
```diff
@@ -207,7 +212,9 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
       if (exitCode === 98) {
         message = nls.localize('force_lightning_lwc_start_addr_in_use');
       }
-
+      if (exitCode === 135){
+        message = nls.localize('force_lightning_lwc_start_no_lwc_folder', 'testPath');
```
Comment (sfsholden) [inline packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts:216]: I think we're good to remove the testPath portion. A customer should see the error message in the terminal if they need more information about the path.

```suggestion
        message = nls.localize('force_lightning_lwc_start_no_lwc_folder');
```

File: packages/salesforcedx-vscode-lwc/src/messages/i18n.ts [hunk 4]
```diff
@@ -29,6 +29,8 @@ export const messages = {
     'The local development server was not able to start.',
   force_lightning_lwc_start_exited:
     'The local development server exited unexpectedly with code %s.',
+  force_lightning_lwc_start_no_lwc_folder:
+    'No \'lwc\' directory found in path %s',
```
Comment (sfsholden) [inline packages/salesforcedx-vscode-lwc/src/messages/i18n.ts:33]:
```suggestion
    'No \'lwc\' directory found in path',
```

Comment (sfsholden) [inline packages/salesforcedx-vscode-lwc/src/messages/i18n.ts:33]:
```suggestion
    'No \'lwc\' directory found',
```

Comment (sfsholden) [reply packages/salesforcedx-vscode-lwc/src/messages/i18n.ts:33]: @ekapner Can you weigh in here for the error message presented to the user?

Context: Customer tries to start code preview without having an lwc folder in their default package directory.

**Comment by lcampos** [comment]
@LisandroFernandezSF is this PR still valid ? If it is can you rebase ? I'm seeing changes on it from other PRs that've been merged already

File: packages/salesforcedx-vscode-lwc/src/messages/i18n.ts (hunk 4 above)
Comment (ekapner) [reply packages/salesforcedx-vscode-lwc/src/messages/i18n.ts]: @lcampos - I didn't see this until now - do I have time to work on the error message?  cc: @LisandroFernandezSF

**Comment by lcampos** [comment]
Closing since this is not needed based on the changes done in lwc dev server

**Pull request closed without merging**